exporters/common/src/main/java/io/opentelemetry/exporter/internal/grpc/GrpcExporter.java

```diff
@@ -62,23 +61,59 @@ public CompletableResultCode export(T exportRequest, int numItems) {
 
     grpcSender.send(
         exportRequest,
-        () -> {
-          exporterMetrics.addSuccess(numItems);
-          result.succeed();
-        },
-        (response, throwable) -> {
-          exporterMetrics.addFailed(numItems);
+        grpcResponse -> {
```

Comment:
I think the readability could be improved by making `onSuccess` and `onError` into functional instance methods.

Reply:
Also applied to the same structure in HttpExporter